assets/js/learners-general.js

```diff
@@ -108,18 +127,12 @@ jQuery(document).ready( function( $ ) {
 				lesson : window.woo_learners_general_data.reset_lesson_confirm,
 				course : window.woo_learners_general_data.reset_course_confirm,
 				action : 'reset_user_post'
-			},
-			remove: {
```

Comment:
I was thinking about that as well. I just put in place a possible solution, but let me know what you think. When they reset progress, this will only start progress back up if they are enrolled. 

2ddf8bd477e54e6369207a16e03c80916fe86ca9

Reply:
Awesome! 🎉